### ISSUE TITLE
NQ - File datasets/nq/qrels/train.tsv not present

### ISSUE THREAD

**Issue by GiacoL** [beir-cellar/beir#179]
I downloaded the NQ dataset and the tsv file for the train set appears to be missing

**Comment by 2020uce0047** [comment]
Hi @GiacoL 
There's another zip file for train set - "nq-train"
All the available datasets - https://public.ukp.informatik.tu-darmstadt.de/thakur/BEIR/datasets/

**Comment by Gerry-j** [comment]
https://public.ukp.informatik.tu-darmstadt.de/thakur/BEIR/datasets/,this url contains nq dataset,but I didn't find train.csv in nq.zip

**Comment by Gerry-j** [comment]
> I downloaded the NQ dataset and the tsv file for the train set appears to be missing

please,where did you finally download train.csv?

**Comment by 2020uce0047** [comment]
Hi @Gerry-j  https://public.ukp.informatik.tu-darmstadt.de/thakur/BEIR/datasets/nq-train.zip

**Comment by Gerry-j** [comment]
Thank you very much！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "beir-cellar/beir"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月15日(星期四) 上午10:31
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [beir-cellar/beir] NQ - File datasets/nq/qrels/train.tsv not present (Issue #179)





 
Hi @Gerry-j  https://public.ukp.informatik.tu-darmstadt.de/thakur/BEIR/datasets/nq-train.zip

**Comment by orionw** [comment]
Hi all, thanks for this info. Is the corpus set not the same between them? I see `18060996` lines in the corpus for this link but the BEIR NQ corpus for test has `2,681,468`?

**Comment by thakur-nandan** [comment]
@orionw yes that's true. The BEIR NQ test corpus is different from the train corpus (much larger, can contain duplicates), therefore for uniformity we choose to not complicate our BEIR structure and provide nq training pairs separately under `nq-train.zip`
